src/CECILL-C-fr.xml

```diff
@@ -291,15 +291,13 @@
                       <bullet>6.4</bullet>
                       MENTIONS DES DROITS
                       <p>Le Licencié s'engage expressément:</p>
-                      <list>
                             <p>à ne pas supprimer ou modifier de quelque manière que ce soit les mentions de propriété
                              intellectuelle apposées sur le Logiciel;</p>
                             <p>à reproduire à l'identique lesdites mentions de propriété intellectuelle sur les copies du
                              Logiciel modifié ou non;</p>
                             <p>à faire en sorte que l'utilisation du Logiciel, ses mentions de propriété intellectuelle et le
                              fait qu'il est régi par le Contrat soient indiqués dans un texte facilement accessible
                              notamment depuis l'interface de tout Logiciel Dérivé.</p> 
-                      </list>
```

Comment:
This looks like it was meant to be a list.  Maybe keep the `<list>` and replace the internal `<p>` with `<item>`?

Reply:
This license is deleted on a subsequent PR #445

Reply:
Fixed in a subsequent commit

Reply:
> This license is deleted on a subsequent PR #445

So land that PR and rebase this one on top?